src/main/scala/com/evolutiongaming/cluster/sharding/ShardingStrategy.scala

```diff
@@ -70,6 +70,61 @@ object ShardingStrategy {
   }
 
 
+  /**
+    * Per-shard allocation and rebalance based on matching shard's "role" with node roles
+    */
+  class FilterByRole(
+    shardRole: Shard => Option[String],
+    toAddress: Region => Address,
+    toGlobal: Address => Address,
```

Comment:
I think proper `toAddress` implementation is enough to drop `toGlobal`

Reply:
And what about regions in `current`?

Reply:
True.

Reply:
Fixed.